### PR TITLE
OAEWDGT-118 - Fix long widget titles breakage

### PR DESCRIPTION
**Pull request by christianvuerings**
Jira: https://jira.sakaiproject.org/browse/OAEWDGT-118

### CODE REVIEW

File: app/views/core/_widget.html.erb [hunk 1]
```diff
@@ -7,13 +7,17 @@
...
-        <% when "medium" %>
-            <%= link_to truncate(widget.title, :length => 20, :omission => "..."), widget_path({:title => widget.url_title}), :class => "wl-regular-link wl-bold wl-widget-item_widgetname", :title => widget.title %>
+        <% if display[:ellipsis] %>
+            <div class="wl-widget-ellipsis"><%= link_to widget.title, widget_path({:title => widget.url_title}), :class => "wl-regular-link wl-bold wl-widget-item_widgetname", :title => widget.title %></div>
         <% else %>
-            <%= link_to truncate(widget.title, :length => 12, :omission => "..."), widget_path({:title => widget.url_title}), :class => "wl-regular-link wl-bold wl-widget-item_widgetname", :title => widget.title %>
+            <% case display[:title_length] %>
+            <% when "long" %>
+                <%= link_to truncate(widget.title, :length => 50, :omission => "..."), widget_path({:title => widget.url_title}), :class => "wl-regular-link wl-bold wl-widget-item_widgetname", :title => widget.title %>
```
Comment (croby) [inline app/views/core/_widget.html.erb]: Should we use these anymore at all?

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -4,6 +4,7 @@
 //= require jquery.remotipart
 //= require modernizr
 //= require jqmodal
+//= require jquery.autoellipsis.js
```
Comment (croby) [inline app/assets/javascripts/application.js]: no need to include `.js` here

Comment (christianvuerings) [inline app/assets/javascripts/application.js]: ok, I'll remove that

File: app/views/core/_widget.html.erb (hunk 1 above)
Comment (christianvuerings) [inline app/views/core/_widget.html.erb]: We still use them in "Recently Added" and "Most Popular".
Should these use ellipsis as well?

If so, I would do something like this:

``` ruby
display[:ellipsis] => 2
```

Where 2 will be the rows that are shown.
In "Recently Added" and "Most Popular" I will set them to 1.

Comment (croby) [inline app/views/core/_widget.html.erb]: sounds good, let's do that

Comment (christianvuerings) [inline app/views/core/_widget.html.erb]: That should be in now.

File: app/assets/javascripts/application.js [hunk 3]
```diff
@@ -45,6 +46,17 @@ $(function() {
         $parentElement.text($parentElement.attr("data-completetext"));
     };
 
-    $("body").on( "click", ".review_read_more", readmore );
+    var addBindings = function() {
+        $( ".wl-page-container" ).on( "change", "#sort_by", performsearch );
```
Comment (croby) [inline app/assets/javascripts/application.js]: I don't think these should be in here, should they?

Comment (christianvuerings) [inline app/assets/javascripts/application.js]: They shouldn't, had missed them when resolving the merge conflicts.
It should be fixed now.

**Pull request merged**